.readthedocs.yaml

```diff
@@ -0,0 +1,24 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.7"
```

Comment:
@cottsay I wonder if it's worth bumping this further up since Python 3.7 is no longer active.

Candidates on the low end: 3.8 (Ubuntu "focal" 20.04), 3.9 (Debian "bullseye" oldstable)
On the high: 3.12 (Ubuntu "noble" 24.04)

Matching OS version: 3.10 (Ubuntu "jammy" 22.04)

Reply:
According to https://blog.readthedocs.com/migrate-configuration-v2/ the original default was 3.7 which was why I chose it. I did have to bump up the ubuntu version from 18.04. It's probably fine to go to 3.10 or 3.11

Reply:
At a glance I'm not seeing any issues after bumping up to 3.11

Reply:
That's fine I suppose. Down the line when we bump to 24.04 can also bump to Python 3.12.

Reply:
Sounds good, thanks.